### DISCUSSION TITLE
Verifying ownership of Namecheap Domain to with Google Search console

### DISCUSSION THREAD

**Discussion by Sowed** [vercel/vercel#4577] [Help] [answered]
Using `Namecheap` domain with Vercel's `now` hosting, I am finding hardship verifying ownership with `Google Search Console`. From [this guide ](https://www.namecheap.com/support/knowledgebase/article.aspx/9252/2208/how-do-i-add-my-domain-to-google-sites), you are required to paste the TXT record into the Advanced DNS `HOST RECORDS`.

Unfortunately this option is unavailable if you setup your `NameServers` to `Custom DNS` which is a requirement for connecting your Namecheap domain to your Vercel deployment, meaning you would have to do the verification from Vercel.

How can I verify the domain ownership then with this flow.?

**Comment by Sowed** [comment]
At the moment, trying to make it work with [Vercel CLI Reference: DNS](https://vercel.com/docs/cli?utm_source=zeit-dashboard&utm_medium=web&utm_campaign=configure-dns#commands/dns).

**Reply by mdandre89** [reply]
you can just do it in your DNS settings of the domain you ported to vercel; just a record of type txt, with the value google gave you; took 5 min

**Comment by mcsdevv** [comment, marked as answer]
Hello @Sowed, as you've mentioned, you need to add the TXT record using the Vercel CLI: https://vercel.com/docs/now-cli#commands/dns

**Reply by Sowed** [reply]
@mcsdevv, I had set it in the CLI like from the docs and confirmed through the [dash](https://vercel.com/dashboard/domains/) that it was set but verification was still failing. It's until after some hours that I checked and it passed. I guess it was a DNS problem.

**Comment by jhinter** [comment]
to save you some time:

1. get vercel cli
`npm i vercel -g`

2. login
`vercel login`

3. create the dns txt record
`vercel dns add example.com '@' 'TXT' 'google-site-verification=XYZ'`

**Reply by ezeBonani** [reply]
thanks a lot! 
in first try it didn't work but in second try it worked perfect!
